.travis.yml

```diff
@@ -36,11 +36,6 @@ env:
           SOS_DISABLE_EXTERNAL_TESTS=1
           SOS_TRANSPORT_OPTS="--with-portals4=$TRAVIS_INSTALL/portals4/"
           SOS_BUILD_OPTS="--enable-pmi-simple"
-        - >
-          SOS_DISABLE_EXTERNAL_TESTS=1
-          SOS_PM="prun -host localhost -oversubscribe" SOS_PM_PRE="prte --daemonize" SOS_PM_POST="prun -terminate"
-          SOS_TRANSPORT_OPTS="--with-portals4=$TRAVIS_INSTALL/portals4/"
-          SOS_BUILD_OPTS="--with-pmix=$TRAVIS_INSTALL/pmix"
         - >
```

Comment:
For Portals, we are removing the PMIx test because of existing issues, right? I am wondering whether we would still keep them commented for any future reasons. Either way is fine though.

Reply:
Ah thanks, that reminds me, I was planning to create an SOS github issue to track it.  Commenting it out would work too, but I have a feeling it might take some time for this to get resolved.  I would argue it's pretty low priority for all parties involved.

Any preference?

Reply:
I would prefer to comment this out. This is a particularly tricky case and it's good to test.